dev/bots/utils.dart

```diff
@@ -426,20 +426,8 @@ Future<void> runDartTest(
   List<String>? tags,
   bool runSkipped = false,
 }) async {
-  int? cpus;
-  final String? cpuVariable = Platform.environment['CPU']; // CPU is set in cirrus.yml
-  if (cpuVariable != null) {
-    cpus = int.tryParse(cpuVariable, radix: 10);
-    if (cpus == null) {
-      foundError(<String>[
-        '${red}The CPU environment variable, if set, must be set to the integer number of available cores.$reset',
-        'Actual value: "$cpuVariable"',
-      ]);
-      return;
-    }
-  } else {
-    cpus = 2; // Don't default to 1, otherwise we won't catch race conditions.
-  }
+  int cpus = Platform.numberOfProcessors;
```

Comment:
I think to be equivalent this should be 2.

If you want to up the parallelism, I'm open to it, but we should just keep an eye out for new flakes caused by this.

Reply:
I would rather not dump that on our team at the moment, so I'll file an issue instead and change this to `2`.

Reply:
SGTM